<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Photo.qrc">
    <normaloff>:/new/prefix1/photo/icon.png</normaloff>:/new/prefix1/photo/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QDateEdit" name="dateEdit_birthday">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
background-image: url(:/new/prefix1/photo/white.png);
color: rgb(26, 222, 213);
border-color: rgb(26, 222, 213);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/photo/color_login.png);
color: rgb(255, 255, 255);
font: 12pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/photo/color_login.png);
color: rgb(255, 255, 255);
font: 12pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/photo/white.png);
font: 12pt &quot;幼圆&quot;;
color: rgb(26, 222, 213);</string>
   </property>
   <property name="text">
    <string>所在地</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/photo/white.png);
font: 12pt &quot;幼圆&quot;;
color: rgb(26, 222, 213);</string>
   </property>
   <property name="text">
    <string> 生日</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_repWord">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
background-image: url(:/new/prefix1/photo/color_login.png);
font: 10pt &quot;幼圆&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/photo/white.png);
color: rgb(21, 216, 224);
border:0.5px groove gray;border-radius:10px;padding:2px 4px;
font: 10pt &quot;幼圆&quot;;
border-color: rgb(26, 222, 213);</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/photo/color_login.png);
color: rgb(255, 255, 255);
font: 12pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_location">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
background-image: url(:/new/prefix1/photo/white.png);
color: rgb(26, 222, 213);
border-color: rgb(26, 222, 213);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/photo/white.png);
font: 12pt &quot;幼圆&quot;;
color: rgb(26, 222, 213);</string>
   </property>
   <property name="text">
    <string> 性别</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
background-image: url(:/new/prefix1/photo/color_login.png);
font: 10pt &quot;幼圆&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pWord">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
background-image: url(:/new/prefix1/photo/color_login.png);
font: 10pt &quot;幼圆&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>365</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/photo/registerbg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/photo/min_close.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/photo/close_blue.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/photo/white.png);
color: rgb(26, 222, 213);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_female">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>女</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonmale">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>男</string>
    </property>
   </widget>
  </widget>
  <zorder>label_7</zorder>
  <zorder>dateEdit_birthday</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit_repWord</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_location</zorder>
  <zorder>label_6</zorder>
  <zorder>lineEdit_name</zorder>
  <zorder>lineEdit_pWord</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_pWord</tabstop>
  <tabstop>lineEdit_repWord</tabstop>
  <tabstop>lineEdit_location</tabstop>
  <tabstop>dateEdit_birthday</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>radioButtonmale_2</tabstop>
  <tabstop>radioButton_female_2</tabstop>
 </tabstops>
 <resources>
  <include location="Photo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Register</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Register</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
